new/omnibus-chef-native-gems.md

```diff
@@ -0,0 +1,49 @@
+---
+RFC: unassigned
+Author: Lamont Granquist <lamont@chef.io>
+Status: Draft
+Type: Standards Track
+---
+
+# Title
+
+Omnibus Chef Native Gem Additions
+
+## Motivation
+
+    As a Chef User,
+    I want to have omnibus-chef ship with nokogiri,
+    Because that will speed up my initial converges, and ease my pain at installing it.
+
+    As a Chef User,
+    I want to have omnibus-chef ship with popular Open Source databases,
+    Because that will speed up my initial converaes, and again ease my pain.
+
+    As a Postgres Cookbook Author,
+    I want to obliterate what we had to do to install Postgres client into omnibus from a cookbook,
+    Because, seriously, have you look at that code?
```

Comment:
*looked, but snark. Unless you wrote all that code and you're being self-deprecating, I don't think we should be negative about it (and thus its authors), even if it was a product of the available Chef features at the time.